pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -8558,7 +8606,7 @@ spec:
 			}
 
 			if len(taskRuns.Items) != 9 {
```

Comment:
```suggestion
			if len(taskRuns.Items) != 10 {
```
Should this be updated to 10?

Reply:
`taskRuns` is retrieving a list of taskRuns created for a `matrix` pipelineTask `platforms-and-browsers` and expected that the reconciler creates 9 taskRuns. I have reverted the changes to the error message to reflect the right number.

If needed, we can change this a little bit more to also retrieve taskRun for the other matrix pipelineTask and make sure it only creates one taskRun.